book/chapters/chapter10/advanced_technical_aspects_of_mlr3.qmd

```diff
@@ -394,22 +394,22 @@ lrn_rpart = lrn("classif.rpart")
 lrn_rpart$train(tsk_sonar)
 
 # set up parallel predict on four workers
-plan("multisession", workers = 4)
+future::plan("multisession", workers = 4)
 lrn_rpart$parallel_predict = TRUE
 
 # predict
-lrn_rpart$predict(tsk_sonar)$score()
+prediction = lrn_rpart$predict(tsk_sonar)
```

Comment:
because this is a "totally technical" example where the results only waste space -- and do not inform the reader about anything. you really gain zero from them -- and the text does not comment on them